packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -222,40 +291,42 @@ export async function findTeamMembersMatchingAttributeLogicOfRoute(
     })
   );
 
-  moduleLogger.debug(
-    "Finding team members matching attribute logic",
-    safeStringify({
-      form,
-      response,
-      routeId,
-      teamId,
-      attributesQueryBuilderConfigFields: attributesQueryBuilderConfig.fields,
-    })
-  );
-
   const [
     teamMembersWithAttributeOptionValuePerAttribute,
     getTeamMembersWithAttributeOptionValuePerAttributeTimeTaken,
   ] = await aPf(() => getTeamMembersWithAttributeOptionValuePerAttribute({ teamId: teamId }));
 
   const logic = getJsonLogic({
-    attributesQueryValue,
+    attributesQueryValue: attributesQueryValue as JsonTree,
     attributesQueryBuilderConfig: attributesQueryBuilderConfig as unknown as Config,
+    isPreview: !!isPreview,
   });
 
   if (!logic) {
     return {
-      teamMembersMatchingAttributeLogic: [],
+      teamMembersMatchingAttributeLogic: null,
```

Comment:
Changed it intentionally to convey that no logic was used to get the resuts like other cases above. 

The behaiour remains same for empty array and null